integration-tests/tests/singlestore/singlestore-common.ts

```diff
@@ -2068,10 +2068,10 @@ export function tests(driver?: string) {
 			const users = await db.select().from(usersTable).orderBy(asc(usersTable.id));
 
 			// check that the timestamps are set correctly for default times
-			expect(Math.abs(users[0]!.createdAt.getTime() - Date.now())).toBeLessThan(2000);
+			expect(Math.abs(users[1]!.createdAt.getTime() - Date.now())).toBeLessThan(2000);
```

Comment:
This is incorrect. The user with default `createdAt` time is the user with `ID=1` and the user with a custom `createdAt` is the second one created (`ID=2`). 
Since the results are ordered by `ID`, it is expected that the first user from the array is the one with the default `createdAt` and the second user from the array is the one with custom `createdAt` value

This test passes

```ts
test('timestamp timezone', async (ctx) => {
			const { db } = ctx.singlestore;

			const date = new Date(Date.parse('2020-01-01T12:34:56+07:00'));

			await db.insert(usersTable).values({ id: 1, name: 'With default times' });
			await db.insert(usersTable).values({
				id: 2,
				name: 'Without default times',
				createdAt: date,
			});
			const users = await db.select().from(usersTable).orderBy(asc(usersTable.id));

			// check that the timestamps are set correctly for default times
			expect(Math.abs(users[0]!.createdAt.getTime() - Date.now())).toBeLessThan(2000);

			// check that the timestamps are set correctly for non default times
			expect(Math.abs(users[1]!.createdAt.getTime() - date.getTime())).toBeLessThan(2000);
		});
	```